docs/resources/css_cluster.md

```diff
@@ -138,19 +152,33 @@ In addition to all arguments above, the following attributes are exported:
 * `created` - Time when a cluster is created. The format is ISO8601:
   CCYY-MM-DDThh:mm:ss.
```

Comment:
suggest to use `hh24`

Reply:
This is a standard iso8601 format. so keep it as it is

Reply:
ok，it should follow the ISO8601 format